.env-example

```diff
@@ -1,3 +1,3 @@
-PUBLICCODE_ELASTIC_URL=http://localhost:9200/indicepa_pec/_search
-PUBLICCODE_REPOSITORY=""
-PUBLICCODE_VALIDATOR_URL=""
+ELASTIC_URL=http://localhost:9200/indicepa_pec/_search
+REPOSITORY=""
```

Comment:
by this init `envsubst` will generate this
`REPOSITORY: """",`
and it is not valid.

Reply:
Hi @sebbalex, this PR actually slightly changes what I initially proposed:

For build via Yarn, `.env` is just loaded by dotenv and it's not modified by `envsubst`.

If `config/appConfig.js` exists, the variables defined there take precedence and that's the case of the Docker images: `envsubst` creates it when the container starts using `appConfig.js.tpl` as template and the environment passed by Docker compose (which in turn uses the very same `.env` file)

Reply:
@sebbalex Fixed with the new pushed version.

Reply:
Thanks!